G/GLFW/build_tarballs.jl

```diff
@@ -1,7 +1,7 @@
 using BinaryBuilder
 
 name = "GLFW"
-version = v"3.3"
+version = v"3.3.2"
```

Comment:
I think you also need to change the source URL and hash :wink:

Reply:
yeah, on it!